tomviz/AboutDialog.cxx

```diff
@@ -24,23 +24,95 @@
 
 #include <pqApplicationCore.h>
 #include <pqSettings.h>
+
+#include "vtkPython.h"
+#include "vtkPythonInterpreter.h"
+#include "vtkSmartPyObject.h"
+#include <vtkNew.h>
 #include <vtkPVConfig.h>
+#include <vtkPVOpenGLInformation.h>
+#include <vtkVersion.h>
+
+#include <QString>
+#include <QTreeWidget>
+
+#include <itkConfigure.h>
 
 namespace tomviz {
 
+static inline void AddItem(QTreeWidget* tree, const QString& name,
+                           const QString& value)
+{
+  QTreeWidgetItem* item = new QTreeWidgetItem(tree);
+  item->setText(0, name);
+  item->setText(1, value);
+}
+
+static inline void AddPythonAttributeItem(QTreeWidget* tree,
+                                          const QString& itemName,
+                                          const char* module,
+                                          const char* attribute)
+{
+  vtkPythonInterpreter::Initialize();
+
+  vtkPythonScopeGilEnsurer gilEnsurer;
+
+  vtkSmartPyObject mod(PyImport_ImportModule(module));
+  if (mod) {
+    vtkSmartPyObject attr(PyObject_GetAttrString(mod, attribute));
+    if (attr) {
+#if PY_MAJOR_VERSION >= 3
+      const char* cdata = PyUnicode_AsUTF8(attr);
+#else
+      const char* cdata = PyBytes_AsString(attr);
+#endif
+      AddItem(tree, itemName, QString::fromStdString(cdata ? cdata : ""));
+    }
+  } else {
+    AddItem(tree, itemName, "unavailable");
+  }
```

Comment:
@cquammen This doesn't compile with Python 3, because Qt defines slots as well as Python 3. I have already created classes in [PythonUtilities] (https://github.com/OpenChemistry/tomviz/blob/master/tomviz/PythonUtilities.h) then encapsulates our python operations, for example you can import a module using this [function](https://github.com/OpenChemistry/tomviz/blob/master/tomviz/PythonUtilities.h#L142). You make have to add some API to do what you want. The ideal is to not leak any python types.